### PR TITLE
INCLUDES SORTING VISUALIZING OF ALL SORTING ALGORITHMS

### LINKED ISSUE #1286: [Feature]: Add Visualization of All sorting algorithms
### Feature Name

Add Visualization of All sorting algorithms 

### Feature Description

Add Visualization of All sorting algorithms so that we can understand easly whats happening and how the algo is work..

### Motivation

we can understand easly whats happening and how the algo is work..

### Implementation Suggestions (Optional)

_No response_

### Feature Type

New Algorithm

### Does this feature require additional resources?

- [ ] Additional Documentation
- [ ] Third-Party Libraries
- [X] UI/UX Design
- [ ] Other

### References (Optional)

_No response_

### PR DESCRIPTION
**Pull request by Meetpidev**
### Description
INCLUDES SORTING VISUALIZING OF ALL SORTING ALGORITHMS

Fixes #1286 

### Type of change
- New feature (non-breaking change which adds functionality)

### Checklist:

- [ YES ] My code follows the style guidelines of this project
- [ YES ] I have performed a self-review of my code
- [ YES ] I have commented my code, particularly in hard-to-understand areas
- [ YES ] I have added tests that prove my fix is effective or that my feature works
- [ YES ] New and existing unit tests pass locally with my changes
- [ YES ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Meetpidev** [comment]
@ajay-dhangar Requested to review

**Comment by Meetpidev** [comment]
@ajay-dhangar Please Review IT..

**Comment by Meetpidev** [comment]
@ajay-dhangar 
### Requested to accept it with level 3 as its includes ALL the sorting algorithm with visualization... we can also change the size of array... we can also manage the speed....

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
Please write your code in the Component folder and use it as a component, not as a page. This will make it easier to maintain and reuse.

**Comment by Meetpidev** [comment]
> Please write your code in the Component folder and use it as a component, not as a page. This will make it easier to maintain and reuse.

Its doen please Review again....

**Comment by Meetpidev** [comment]
> > Please write your code in the Component folder and use it as a component, not as a page. This will make it easier to maintain and reuse.
> 
> Its done please Review again....

**Comment by Meetpidev** [comment]
@ajay-dhangar I replace code in component folder... please review it

**Comment by Meetpidev** [comment]
@ajay-dhangar Please Review PR..

File: docusaurus.config.js [hunk 1]
```diff
@@ -104,6 +104,11 @@ const config = {
             label: "Top DSA Interview Questions",
             position: "left",
           },
+          {
+            to: "Visualizing",
+            label: "Sorting Visualizer",
+            position: "left",
+          },
```
Comment (ajay-dhangar) [inline docusaurus.config.js]: delete lines `107` to `111`

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
We are not using a link for the url and pages, so can you delete the code based on the suggestions

File: docusaurus.config.js (hunk 1 above)
Comment (Meetpidev) [reply docusaurus.config.js:111]: so how can i defined that new feature ? in navbar i've addedthis..

**Comment by Meetpidev** [comment]
@ajay-dhangar I removed it.. please Review again...

File: docusaurus.config.js (hunk 1 above)
Comment (Meetpidev) [reply docusaurus.config.js]: I removed it.. please Review Again...

Comment (Meetpidev) [reply docusaurus.config.js]: i removed it but when should i render it ? should i render it in home page ?

Comment (Meetpidev) [reply docusaurus.config.js]: requested to guid me

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**